arch/otbn/printer.s.dfy

```diff
@@ -403,23 +404,16 @@ class Printer {
                     print(".globl "); print(func_name); print("\n");
                 }
                 print(func_name); print(":\n");
+                var overlaps := has_overlap_seq(simplify_codes(res[i].1));
+                if overlaps {
+                  print("LOOP_ERROR detected:\n");
+                  print(simplify_codes(res[i].1));
+                }
                 printCode(Block(res[i].1));
                 printIndent(); print("ret\n\n");
             }
             i := i + 1;
         }
     }
 }
-
-method Main()
-{
-    var comment := "/*\n  This code is generated by the veri-titan project: https://github.com/secure-foundations/veri-titan\n*/\n";
```

Comment:
This comment seems like it might be useful to retain.  Can we add it to `printTopLevelProc`?

Reply:
Sure, actually moved it into to `modexp_printer`.